### ISSUE TITLE
slideTo, slideNext() ignores "effect" parameter?

### ISSUE THREAD

**Issue by yuchant** [nolimits4web/Swiper#1386]
With "fade" set as the effect parameter, slideTo seems to ignore it.

How do we programmatically go to the next slide using the fade effect?

Thank you!

**Comment by nolimits4web** [comment]
Are you sure about that? Check the demo http://www.idangero.us/swiper/demos/16-effect-fade.html . Does it ignore effect when you click on arrows?

**Comment by yuchant** [comment]
Ah, thank you! After hearing it should work, I can reproduce it only if I have my slide content wrapper with the `swiper-no-swiping` class.

I'm using the swiper as a non-user swipeable cross fader (multi step form). Disabling swiping prevents programmatic swiping at all, so I have wrapped the inside of my slides in a `swiper-no-swiping` class, so that I can programmatically slide through the slides only.

`<div class="swiper-slide"><div class="swiper-no-swiping">...</div></div>`

Any suggestions?

**Comment by nolimits4web** [comment]
Do you have a live example or JSFiddle?

**Comment by yuchant** [comment]
Hey Vladimir,

I just tried to reproduce it on jsfiddle, and could not.

I noticed something interesting though... the transition only happens on
the first slide. The next transitions work fine.

This likely meant the initial transforms were simply wrong because the
swiper is initialized while hidden. Passing true to the `update` function
as described in the docs solved the problem.

Thank you for walking through this with me.

The library is a game changer.
